spec/services/default_form_builder_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe DefaultFormBuilder do
+  it 'creates a default master form' do
+    expect{
+      DefaultFormBuilder.create
+    }.to change{ Form.count }.from(0).to(1)
+  end
+
+  it 'creates fields' do
+    expect( DefaultFormBuilder.create.form_elements.size ).to eq(3)
+  end
+
+  context 'when form already exists' do
+    before {  DefaultFormBuilder.create  }
+
+    it 'returns form' do
+      expect {
+         DefaultFormBuilder.create
```

Comment:
this is a good test but doesn't really test what it says it does. maybe turn this into a `returns form` test and a `doesn't create a new form` test?